src/test/java/com/foundation/salesforce/steps/ContactAcceptanceSteps.java

```diff
@@ -0,0 +1,268 @@
+     * Method to let a build a new contact.
+     *
+     * @param inputContent
+     */
+    @Given("user specifies new contact$")
+    public void userCreateNewContact(Map<String, String> inputContent) {
+        RestClientApi.getInstance().buildSpec(inputContent);
+    }
+
+    /**
+     * Method to let user sent a request.
+     */
+    @When("User send de request post to contact endpoint")
+    public void userSendDeRequestPostToContactEndpoint() {
+        response = RestClientApi.getInstance().post(EndPoints.CONTACT_ENDPOINT);
+    }
+
+    /**
+     * Method to let compare a recuest.
+     *
+     * @param statusCode
+     */
+    @Then("User get a {string} status code")
+    public void userGetAStatusCode(String statusCode) {
+        Assert.assertEquals(response.getStatusCode(), Integer.parseInt(statusCode));
+    }
+
+    /**
+     * Method to led a specifies a new contact.
+     *
+     * @param inputContent
+     */
+    @Given("User specifies new contact with firstName")
+    public void userSpecifiesNewContactWithFirstName(Map<String, String> inputContent) {
+        RestClientApi.getInstance().buildSpec(inputContent);
+    }
+
+    /**
+     * Method to let a sent a request to contact.
+     */
+    @When("User send de request post to contact endpoint with firstName")
+    public void userSendDeRequestPostToContactEndpointWithFirstName() {
+        response = RestClientApi.getInstance().post(EndPoints.CONTACT_ENDPOINT);
+    }
+
+    /**
+     * Method let me get the status code with first name.
+     *
+     * @param statusCode
+     */
+    @Then("User get a {string} status code with firstName")
+    public void userGetAStatusCodeWithFirstName(String statusCode) {
+        Assert.assertEquals(response.getStatusCode(), Integer.parseInt(statusCode));
+    }
+
+    /**
+     * Method to let me specifies new contact with the first name and email.
+     *
+     * @param inputContent
+     */
+    @Given("User specifies new contact with firstName, email")
+    public void userSpecifiesNewContactWithFirstNameEmail(Map<String, String> inputContent) {
+        RestClientApi.getInstance().buildSpec(inputContent);
+    }
+
+    /**
+     * Mehto let send a recuest to contact with first name and email.
+     */
+    @When("User send de request post to contact endpoint with firstName, email")
+    public void userSendDeRequestPostToContactEndpointWithFirstNameEmail() {
+        response = RestClientApi.getInstance().post(EndPoints.CONTACT_ENDPOINT);
+    }
+
+    /**
+     * Method to led me get a status code with first name and email.
+     *
+     * @param statusCode
+     */
+    @Then("User get a {string} status code with firstName, email")
+    public void userGetAStatusCodeWithFirstNameEmail(String statusCode) {
+        Assert.assertEquals(response.getStatusCode(), Integer.parseInt(statusCode));
+    }
+
+    /**
+     * Method to let user specifies new contact with first name, email and title.
+     *
+     * @param inputContent
```

Comment:
Please, place descriptions for the parameters.

Reply:
done.